products/ipados.md

```diff
@@ -11,11 +11,16 @@ releaseDateColumn: true
 auto:
 -   custom: true
 releases:
+-   releaseCycle: "16"
+    eol: false
+    releaseDate: 2022-10-24
```

Comment:
releaseDate looks like ( for 16,x series ) September 12, 2022  according to wikipedia
[we are using releaseDate tag for the first release version of related cycle ]